lib/electric_slide/call_queue.rb

```diff
@@ -216,9 +216,10 @@ def return_agent!(*args)
 
     # Removes an agent from the queue entirely
     # @param [Agent] agent The {Agent} to be removed from the queue
+    # @param [extra_params] hash Application specific extra params
```

Comment:
Ops! :P

I think I got confused on the above line `[Agent] agent`, but now it makes more sense :)

Thanks!